README.rst

```diff
@@ -1,6 +1,12 @@
 OpenSoar
```

Comment:
Determine new version number (backwards incompatible maybe?) and write changelog